build.sbt

```diff
@@ -1,6 +1,6 @@
 inThisBuild(
   List(
-    organization := "com.geirsson",
+    organization := "org.scalameta",
```

Comment:
The org change needs to be mirrored here as well:

* `plugin/src/sbt-test/scalafmt-sbt/sbt-cli/project/plugins.sbt`
* `plugin/src/sbt-test/scalafmt-sbt/sbt/project/plugins.sbt`